<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>683</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>683</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PySide2 QMenuBar().</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../images/icons/icon.png</normaloff>../../../../images/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="action_sair_1"/>
    <addaction name="action_sair_2"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="action_sobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_sair_1">
   <property name="text">
    <string>Sair 1</string>
   </property>
  </action>
  <action name="action_sair_2">
   <property name="icon">
    <iconset>
     <normaloff>../../../../images/icons/exit-64x64.png</normaloff>../../../../images/icons/exit-64x64.png</iconset>
   </property>
   <property name="text">
    <string>Sair 2</string>
   </property>
  </action>
  <action name="action_sobre">
   <property name="text">
    <string>Sobre este app</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
